Sources/CLI/Volume/VolumeCreate.swift

```diff
@@ -0,0 +1,63 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the container project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import ContainerClient
+
+extension Application.VolumeCommand {
+    struct VolumeCreate: AsyncParsableCommand {
+        static let configuration = CommandConfiguration(
+            commandName: "create",
+            abstract: "Create a volume"
+        )
+
+        @Argument(help: "Volume name")
+        var name: String
+
+        @Option(name: .long, help: "Volume driver name")
+        var driver: String = "local"
+
+        @Option(name: .customLong("opt"), parsing: .upToNextOption, help: "Set driver specific options")
+        var driverOpts: [String] = []
+
+        @Option(name: .customLong("label"), parsing: .upToNextOption, help: "Set metadata on a volume")
+        var labels: [String] = []
+
+        func run() async throws {
+            print("TODO: Create volume '\(name)' with driver '\(driver)'")
+
+            // Parse driver options
+            let parsedDriverOpts = parseKeyValuePairs(driverOpts)
+            let parsedLabels = parseKeyValuePairs(labels)
+
+            print("Driver options: \(parsedDriverOpts)")
+            print("Labels: \(parsedLabels)")
+        }
+
+        private func parseKeyValuePairs(_ pairs: [String]) -> [String: String] {
```

Comment:
Extract into ContainerClient utility and write a unit test? This is very generic functionality (`--network` could add key-value opts), and might also exist elsewhere already.